Dockerfile

```diff
@@ -1,30 +1,20 @@
-# using rocker r-vers as a base with R 4.3.1
-# https://hub.docker.com/r/rocker/r-ver
-# https://rocker-project.org/images/versioned/r-ver.html
-
-# set proper base image
-ARG R_VERS="4.3.1"
-FROM rocker/r-ver:$R_VERS AS base
+FROM docker.io/rocker/r-ver:4.3.1 AS base
 
 # set Docker image labels
 LABEL org.opencontainers.image.source=https://github.com/RMI-PACTA/workflow.pacta
-LABEL org.opencontainers.image.description="Docker image to run PACTA"
+LABEL org.opencontainers.image.description="Docker image to run PACTA analysis"
 LABEL org.opencontainers.image.licenses=MIT
-LABEL org.opencontainers.image.title=""
-LABEL org.opencontainers.image.revision=""
-LABEL org.opencontainers.image.version=""
-LABEL org.opencontainers.image.vendor=""
-LABEL org.opencontainers.image.base.name=""
-LABEL org.opencontainers.image.ref.name=""
-LABEL org.opencontainers.image.authors=""
+LABEL org.opencontainers.image.title="workflow.pacta"
+LABEL org.opencontainers.image.vendor="RMI"
+LABEL org.opencontainers.image.base.name="docker.io/rocker/r-ver:4.3.1"
+LABEL org.opencontainers.image.authors="Alex Axthelm, CJ Yetman, Jackson Hoffart"
 
 # set apt-get to noninteractive mode
-ARG DEBIAN_FRONTEND="noninteractive"
-ARG DEBCONF_NOWARNINGS="yes"
```

Comment:
you might end up with RED warning messages that are rather incomprehensible without this

Reply:
The DEBIAN FRONTEND just got moved a few lines down.

The DEBCONF Warnings haven't been an issue so far.